templates/components/common/alert/alert-info.html

```diff
@@ -1,4 +1,4 @@
-<div class="alertBox alertBox--info">
+<div class="alertBox alertBox--info" data-marketing-banner>
```

Comment:
Could we change data attribute to data-promotion-banner?

Reply:
done